lib/assets/stylesheets/overrides.scss

```diff
@@ -544,3 +544,26 @@ $tooltip-background: $grey;
   border: none;
   background: none;
 }
+
+/* Fieldset styling */
+fieldset.project-details {
```

Comment:
Is there a need to override fieldset and legend just for project-details?
If not it would be nice to see this genericly in dmproadmap.scss as changes to overrides.scss require merging with custom branding overrides.

Reply:
Well the fieldsets with borders are requested only for the project details page currently. The other places where we are using fieldsets are for grouping radio buttons or checkboxes which have no border styling. That is the reason why this is applicable only to this  page. I guess I could move this code to dmproadmap.scss but it would still pally only to project details page.

Reply:
@bhavi did all of her Bootstrap work in the overrides.scss file. I think the stuff in the dmproadmap.scss was added by @vyruss at one point for the header and footer changes. I think reorganizing the CSS files is beyond the scope of this ticket.